portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/clay/ClayAlertToast.testcase

```diff
@@ -97,6 +97,31 @@ definition {
 		}
 	}
 
+	@description = "LPS-170577 - Verify toast alert with action button disappears after 10 seconds"
+	@priority = "5"
+	test DisappearsAfter10SecsWithActions {
+		task ("When a toast alert appears") {
+			Click(
+				key_text = "Disappears After 10 Seconds",
+				locator1 = "Button#ANY");
+		}
+
+		task ("And there is an action button on it") {
+			VerifyElementPresent(
+				key_tag = "//a",
+				locator1 = "ClayAlertToast#TOAST_ALERT_10_SECONDS_TAG");
+		}
+
+		task ("Then the alert disappears after 10 secs") {
+
+			// For LPS-170577, need this pause to see if after 10 seconds, the alert is not showing anymore
+
+			Pause(locator1 = "10000");
+
+			AssertElementNotPresent(locator1 = "ClayAlertToast#TOAST_ALERT_10_SECONDS");
```

Comment:
When you run the test locally, does the test hang for a few seconds on this assertion? Or does the test immediately assert right after the 10 second pause? There may be a built in waitFor within AssertElementNotPresent that we don't want.

Reply:
ok, I'll check it